src/expr/proof_checker.cpp

```diff
@@ -194,17 +161,20 @@ Node ProofChecker::checkDebug(PfRule id,
   // since we are debugging, we want to treat trusted (null) checkers as
   // a failure.
   Node res = checkInternal(id, cchildren, args, expected, out, false);
-  Trace(traceTag) << "ProofChecker::checkDebug: " << id;
-  if (res.isNull())
-  {
-    Trace(traceTag) << " failed, " << out.str() << std::endl;
-  }
-  else
+  if (Trace.isOn(traceTag))
```

Comment:
Update the documentation of `checkDebug` to say that the check is done if we give an active trace.

Reply:
Actually this was just an optimization, the traceTag doesn't impact whether the check is run (usually you are looking to see the return value).